gffquant/profilers/feature_quantifier.py

```
@@ -192,15 +205,19 @@ def register_reference(self, rid, aln_reader):
+        aln_reader: AlignmentProcessor,
+        sam_prefix="",
         min_identity=None,
         min_seqlen=None,
         unmarked_orphans=False,
+        debug_samfile=None,
+        # panda: PandaProfiler=None,
     ):
         # pylint: disable=R0914
         t0 = time.time()
 
+        samfile = f"{self.out_prefix}{sam_prefix}.filtered.sam" if self.debug else None
```

Ruff (0.8.0)

Local variable `samfile` is assigned to but never used

Remove assignment to unused variable `samfile`

(F841)